.pre-commit-config.yaml

```
@@ -1,79 +1,71 @@
 ---
 # https://github.com/antonbabenko/pre-commit-terraform
-# brew install pre-commit terraform-docs tflint tfsec trivy checkov terrascan infracost tfupdate minamijoyo/hcledit/hcledit jq
-
 # https://github.com/pre-commit/pre-commit-hooks
 # https://github.com/python-jsonschema/check-jsonschema/
 
-# https://github.com/trufflesecurity/trufflehog
+fail_fast: true
 
 repos:
 - repo: https://github.com/compilerla/conventional-pre-commit
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 1

(indentation)